### DISCUSSION TITLE
Is there a way to sync the projects.json?

### DISCUSSION THREAD

**Discussion by frontendbaer** [alefragnani/vscode-project-manager#694] [Q&A] [answered]
Hi,

with the sync option in VS the json is not synced, is there another possibility?

**Comment by alefragnani** [comment, marked as answer]
Hi @frontendbaer ,

Not yet, but it would be in the future using Gists. It is being tracked here https://github.com/alefragnani/vscode-project-manager/issues/525

Hope this helps
